_schema.graphql

```diff
@@ -12,6 +12,9 @@ type Collection {
   # Description of the collection which can include links to other collections
   description: String
 
+  # Markdown alternate of description field contents.
+  descriptionMarkdown: String
+
```

Comment:
Is there a reason why this is in this PR? Shouldn't this already be in-place in kaws?

Reply:
Nah, I think the `dump-schema` was just omitted from #186 (including it there would have been preferable, but not a dealbreaker since the dumped schema is mainly for MP's benefit, not Kaws')

I just snuck it in here since I'm in Kaws anyway

Reply:
oooooh, I didn't check in that file - I assumed it was generated and got blown away. That makes sense, sorry about that!